index.js

```diff
@@ -391,6 +391,37 @@
       [$.Function, b, a, c],
       function(f, x, y) { return f(y, x); });
 
+  //# map :: Functor f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a function and a [Functor][], applies the function to each of the
+  //. functor's values, and returns a functor of the same shape.
+  //.
+  //. ```javascript
+  //. > S.map(S.mult(4), S.Just(2))
```

Comment:
I'd prefer to use `S.inc` here rather than `S.mult(4)`, to reduce the cognitive load slightly.